src/App.tsx

```diff
@@ -1,34 +1,108 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState, useCallback } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { FilterStatus } from './types/FilterStatus';
+import { getTodos, getUser } from './api';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
-export const App: React.FC = () => {
+export const App = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(true);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(true);
```
boolean should start with is has

Also it's strange that default value is true, better start with false and enable it where needed